### PR TITLE
Update project filters

### PR DESCRIPTION
**Pull request by koffeinfrei**

### CODE REVIEW

File: app/services/git_repository_cloner.rb [hunk 1]
```diff
@@ -33,32 +33,36 @@ def clone_or_update(user, password, repositories, clone_url_method)
...
-      else
-        puts "Cloning #{repository.name}"
-        Rugged::Repository.clone_at(
-          remote_url,
-          target_directory,
-          bare: true,
-          credentials: credentials
-        )
+      rescue => e
+        $stderr.puts "--> Cloning / Updating of #{repository.html_url} Failed: #{e}"
```
Comment (toupeira) [inline app/services/git_repository_cloner.rb]: Use a more specific exception if it makes sense

File: config/schedule.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 set :job_template, "nice ionice -c3 bash -l -c ':job'"
-set :output, { standard: nil }
+set :output, { standard: nil, error: 'log/cron.stderr.log' }
```
Comment (toupeira) [inline config/schedule.rb]: Nice! Added to https://git.panter.ch/panter/wiki/wikis/hosting/ruby#cronjob-mails :)

File: app/services/git_repository_cloner.rb (hunk 1 above)
Comment (koffeinfrei) [reply app/services/git_repository_cloner.rb]: This should just recover from any `StandardError` actually. I just want to make sure that the cloning process doesn't bail on error.

Comment (toupeira) [reply app/services/git_repository_cloner.rb]: Ah ok, didn't realize `rescue => e` only catches `StandardError`'s :)

**Pull request merged**